src/charactersheet/viewmodels/common/player_card/index.html

```diff
@@ -7,20 +7,35 @@
       <!-- Player Info -->
       <div class="row">
         <div class="col-xs-12">
-          <img class="img-circle center-block"
-            data-bind="
-              css: {
-                'patron-circle': player.isActivePatron,
-                [player.canonicalPatreonTier]: player.isActivePatron,
-              },
-              attr: {
-                src: player.imageUrl || 'https://www.gravatar.com/avatar/{}?d=mm',
-                height: 80,
-                width: 80,
-                alt: player.isActivePatron ? `${player.name} is an active patron.` : '',
-              },
-            "
+          <a
+            role="button"
+            tabindex="0"
+            class="statsPopover"
+            data-bind="popover: {
+              placement: 'bottom',
+              content: (
+                player.isActivePatron
+                ? `${player.name || 'Player'} is a <a href='https://www.patreon.com/adventurerscodex' target='_blank'>supporter</a> on Patreon.`
```

Comment:
Do you think it'd be good to add what `tier` supporter they are? so it'd be
> Tany is a Gold supporter

?

Reply:
I thought about that, but removed it. I cannot remember why, but it was a choice.

Reply:
Are you opposed to adding it back? I feel like it could be a sort of status symbol/flex thing. The goal of the badge and text is to attract others to become patrons.

Reply:
I have no preference. The only limitation is that our API assumes the name of the tiers, so we just need to always ensure that the syntax for our tiers is `{metal} Tier`.

Reply:
which shouldnt be a problem, right?

Reply:
No, just a consequence of the current design.